src/main/java/com/cdancy/jenkins/rest/filters/JenkinsAuthenticationFilter.java

```diff
@@ -39,8 +39,9 @@
     private final JenkinsApi jenkinsApi;
 
     // key = Crumb, value = true if exception is ResourceNotFoundException false otherwise
-    private static volatile Pair<Crumb, Boolean> crumbPair = null; 
+    private static volatile Pair<Crumb, Boolean> crumbPair = null;
     private static final String CRUMB_HEADER = "Jenkins-Crumb";
+    private static final String CRUMB_HEADER_EXPAND = ".crumb";
```

Comment:
This will be very hard to understand without more context. Can you please open an Issue where your explain the problem encountered (perhaps with a dump of the http transaction comparing 1.6.x with 2.x using curl -v), and make this a separate pull-request?

@cdancy, is my request reasonable?